backend/onedrive/quickxorhash/xor_1.20.go

```diff
@@ -0,0 +1,9 @@
+//go:build 1.20 || 1.21 || 1.22
```

Comment:
This build constraint should say just

    //go:build go1.20

I think. That means go 1.20 or later.